apollo-router/src/configuration/experimental.rs

```diff
@@ -123,6 +123,9 @@ mod tests {
                 "sub": {
                     "experimental_trace_id": "ok"
                 }
+            },
+            "preview_subscription": {
```

Comment:
This should probably not be here.

Reply:
It's a test for experimental, but I agree, I'll delete it in a followup PR